packages/app/src/components/MyPlan/Routine/HistoryModal.tsx

```diff
@@ -0,0 +1,28 @@
+import { colors } from 'design-token';
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { ScrollView } from 'react-native-gesture-handler';
+import { Button, Text } from 'ui';
+import TaskItem from './TaskItem';
+
+const HistoryModal = () => {
+  const historyData: any[] = [{}, {}, {}, {}, {}, {}];
```

Comment:
```suggestion
  const historyData: any[] = Array.from({ length: 6 });
```

빈 배열 생성할 때 이렇게 하면 편합니다..!

Reply:
꿀팁 감사합니닷!